openvpn/ansible/main.yaml

```
@@ -0,0 +1,16 @@
+---
+- hosts: localhost
+  tasks:
+    - name: Ensure directories are created
+      loop:
+        - "{{ rootfs }}"
+        - "{{ rootfs }}/etc"
+        - "{{ rootfs }}/etc/openvpn"
+      ansible.builtin.file:
+        path: "{{ item }}"
+        state: directory
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)